packages/app/browser/src/app.html

```diff
@@ -8,21 +8,20 @@
 
 <body>
 	<div class="login">
-		<div class="back">
-			<- Back </div>
-			<h4 class="title">code-server</h4>
-				<h2 class="subtitle">
-					Enter server password
-				</h2>
-				<div class="mdc-text-field">
-					<input type="password" id="password" class="mdc-text-field__input" required>
-					<label class="mdc-floating-label" for="password">Password</label>
-					<div class="mdc-line-ripple"></div>
-				</div>
-				<button id="submit" class="mdc-button mdc-button--unelevated">
-					<span class="mdc-button__label">Enter IDE</span>
-				</button>
+		<div class="back"> <- Back </div>
```

Comment:
Glad you fixed the indentation here, but it did make the diff a little harder to read. The only thing you changed here is to add the error display div right?

Reply:
Yep, just added the error display element.